neo/Consensus/ConsensusService.cs

```diff
@@ -280,16 +280,17 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
             isRecovering = true;
             try
             {
-                if (message.ViewNumber > context.ViewNumber)
-                {
-                    if (context.CommitSent()) return;
-                    ConsensusPayload[] changeViewPayloads = message.GetChangeViewPayloads(context, payload);
-                    foreach (ConsensusPayload changeViewPayload in changeViewPayloads)
-                        ReverifyAndProcessPayload(changeViewPayload);
-                }
-                if (message.ViewNumber != context.ViewNumber) return;
                 if (!context.CommitSent())
                 {
+                    if (message.ViewNumber > context.ViewNumber)
+                    {
+                        if (context.CommitSent()) return;
```

Comment:
In line 283 you have the opposite verification

Reply:
Yessss, sorry for that, we just removed line 283. You and Jeff suggested at the same time...aehauhea